solvcon/parcel/gas/inout.py

```diff
@@ -65,12 +65,27 @@ def __init__(self, cse, show_bclist=False, perffn=None, **kw):
         super(MeshInfoHook, self).__init__(cse, **kw)
 
     def preloop(self):
+        flag_generic_bc = False
         blk = self.blk
         self.info("Block information:\n  %s\n" % str(blk))
         if self.show_bclist:
             for bc in blk.bclist:
                 self.info("  %s\n" % bc)
 
+        # explicit info to prompt users to watch out issues like issue #177
+        for bc in blk.bclist:
+            if bc.__class__ is sc.BC:
+                flag_generic_bc = True
+        if flag_generic_bc:
```

Comment:
Thanks for the comment. Using the built-in function is indeed a better way. Fixed.